src/script/hooks/useAppSoftLock.ts

```diff
@@ -40,12 +40,11 @@ export function useAppSoftLock(callingRepository: CallingRepository, notificatio
 
   useEffect(() => {
     const e2eiHandler = E2EIHandler.getInstance();
-    if (!e2eiHandler.isE2EIEnabled()) {
-      return () => {};
-    }
 
+    e2eiHandler.on('initialized', handleSoftLockActivation);
```

Comment:
Here we make sure that we compute the lock state as soon as the handler is initialized